### PR TITLE
Feature/cert path

### PR DESCRIPTION
**Pull request by quentingodeau**
To simplify the development at my side I have continue on the *feature/multi-sa* branch that is already in an other pull request, if needed I can always rebase the branch.
Regards,
Quentin

### CODE REVIEW

**Comment by quentingodeau** [comment]
Here I have two possibilities, I can set the version explicitly in the *vcpkg.json* or create a PR for the duckdb repo to update the baseline.json (that is included with the vcpkg repo).
What do you think is the best way to handle this situation?

**Comment by samansmink** [comment]
> Here I have two possibilities, I can set the version explicitly in the _vcpkg.json_ or create a PR for the duckdb repo to update the baseline.json (that is included with the vcpkg repo). What do you think is the best way to handle this situation?

Yea just bumping in this extension is fine. Note that for the `Main Extension Distribution Pipeline` pipeline there is an input param `vcpkg_version` to configure that one. Eventually we will bump the version in duckdb, then when we start building the azure extension for latest master instead of latest stable, we can remove the vcpkg_version input again!

File: src/azure_secret.cpp [hunk 1]
```diff
@@ -10,29 +11,25 @@
 #include <azure/storage/blobs.hpp>
 
 namespace duckdb {
+constexpr auto COMMON_OPTIONS = {
+    // Proxy option
+    "http_proxy", "proxy_user_name", "proxy_password",
+    // Storage account option
+    "account_name", "transport_option_type"};
```
[the remark this answers is not in the document]
Comment (quentingodeau) [reply src/azure_secret.cpp:18]: Yes i think it make more sens as a setting I will do the changes

[the remark this answers is not in the document]
Comment (quentingodeau) [reply src/azure_secret.cpp]: Done in [79bfd25](https://github.com/duckdb/duckdb_azure/pull/35/commits/79bfd256ceea3515e6c4ca9f66fac5464b81e4c1)

**Comment by quentingodeau** [comment]
> > Here I have two possibilities, I can set the version explicitly in the _vcpkg.json_ or create a PR for the duckdb repo to update the baseline.json (that is included with the vcpkg repo). What do you think is the best way to handle this situation?
> 
> Yea just bumping in this extension is fine. Note that for the `Main Extension Distribution Pipeline` pipeline there is an input param `vcpkg_version` to configure that one. Eventually we will bump the version in duckdb, then when we start building the azure extension for latest master instead of latest stable, we can remove the vcpkg_version input again!

Ok i miss the fact it was a input param of the extension distribution.

**Comment by samansmink** [comment]
Hey @quentingodeau thanks again! i'm sure this fix will make a lot of people happy :)

**Comment by quentingodeau** [comment]
@samansmink thx a lot for all the reviews!

**Pull request merged**